pyvista/plotting/plotting.py

```diff
@@ -3344,7 +3344,7 @@ def add_volume(
             Block resolution. For example ``[1, 1, 1]``. Resolution must be
             non-negative. While VTK accepts negative spacing, this results in
             unexpected behavior. See:
-            `pyvista #1967 <https://github.com/pyvista/pyvista/issues/1967>`_
+            `pyvista #1967 <https://github.com/pyvista/pyvista/issues/1967>`.
```

Comment:
We need the underscore I think  https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages

```suggestion
            `pyvista #1967 <https://github.com/pyvista/pyvista/issues/1967>`_.
```